### PR TITLE
Adding nxos_acl_interface_module

### PR DESCRIPTION
**Pull request by GGabriele**
##### ISSUE TYPE
- New Module Pull Request
##### COMPONENT NAME

nxos_acl_interface
##### ANSIBLE VERSION

```
ansible 2.2.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/usr/share/ansible/']
```
##### SUMMARY

New module to apply ACLs to interfaces.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks again to @GGabriele. This module is going into community review.

We notice that you are one of the maintainers (@privateip, @GGabriele, @gundalow) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

File: network/nxos/nxos_acl_interface.py [hunk 1]
```diff
@@ -0,0 +1,537 @@
...
+    - Jason Edelman (@jedelman8)
+    - Gabriele Gerbino (@GGabriele)
+options:
+    name:
+        description:
+            - Case sensitive name of the access list (ACL).
+        required: true
+    interface:
+        description:
+            - Full name of interface. MUST be the full name.
```
Comment (gundalow) [inline network/nxos/nxos_acl_interface.py]: full name, e.g. I(give an example).

File: network/nxos/nxos_acl_interface.py [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+            module.cli.add_commands(commands, output='config')
+            module.cli.run_commands()
+        except ShellError:
+            clie = get_exception()
+            module.fail_json(msg='Error sending CLI commands',
+                             error=str(clie), commands=commands)
+
+
+def main():
+    argument_spec = dict(
```
Comment (gundalow) [inline network/nxos/nxos_acl_interface.py:455]: As stated in other reviews, please split `argument_spec` so only the specific options defined in the module are listed here.

File: network/nxos/nxos_acl_interface.py [hunk 3]
```diff
@@ -0,0 +1,537 @@
...
+    - Jason Edelman (@jedelman8)
+    - Gabriele Gerbino (@GGabriele)
+options:
+    name:
+        description:
+            - Case sensitive name of the access list (ACL).
+        required: true
+    interface:
+        description:
+            - Full name of interface, e.g. I(Ethernet1/1)
```
Comment (gundalow) [inline network/nxos/nxos_acl_interface.py]: Missing fullstop

**Pull request merged**